lib/units.scss

```diff
@@ -10,3 +10,10 @@ $mmf-height-label: 1.1em !default;
 // animation
 $mmf-transition--time: 0.1s;
 $mmf-transition: $mmf-transition--time linear;
+
+// text & line styles
+$mmf-line-height: 140.62%;
```

Comment:
Are you sure, you want percent in line-heights? em are currently used as default (and should behave the same as %). IMHO, em is mor readable, as it references the font-size at the current location, whereas % is a little less specific. Using `1.4062em` should result in same line-height, but shows us, that the value might be a little too specific.